core/rawdb/accessors_indexes_test.go

```diff
@@ -56,7 +56,9 @@ func TestLookupStorage(t *testing.T) {
 						Index:      uint64(index),
 					}
 					data, _ := rlp.EncodeToBytes(entry)
-					db.Put(txLookupKey(tx.Hash()), data)
+					if err := db.Put(txLookupKey(tx.Hash()), data); err != nil {
+						// ignore error
+					}
```

Comment:
It'd be good to check the error from `rlp.EncodeToBytes` one line up, but it's OK to ignore the error from `db.Put`.